webchanges/reporters.py

```diff
@@ -621,24 +621,37 @@ class StdoutReporter(TextReporter):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
         self._has_color = sys.stdout.isatty() and self.config.get('color', False)
+        color_map = {
+            'default': 39,
+            'black': 30,
+            'red': 31,
+            'green': 32,
+            'yellow': 33,
+            'blue': 34,
+            'magenta': 35,
+            'cyan': 36,
+            'lightgray': 37,
```

Comment:
This color is called 'white' in the ANSI standard (see https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit)